Formula/embree.rb

```diff
@@ -17,7 +17,11 @@ class Embree < Formula
   depends_on "tbb"
 
   def install
-    max_isa = MacOS.version.requires_sse42? ? "SSE4.2" : "SSE2"
+    max_isa = "SSE4.2"
+
+    on_macos do
+      max_isa = "SSE2" unless MacOS.version.requires_sse42?
+    end
```

Comment:
I think we require Nehalem as a minimal CPU, which would make this useless. Someone from @Homebrew/brew probably knows that better.

Reply:
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/os/mac/hardware.rb

Reply:
So in core the minimal supported CPU will always support sse4.2

Reply:
It won't work if you're still on High Sierra, though. I know we don't support High Sierra, but I also think we shouldn't knowingly break things for users still on it, especially when the accommodation is minor.

Reply:
I agree that this is a sufficiently small accommodation that it probably doesn't hurt to leave it in.